### ISSUE TITLE
Wrong "Warning" on Approved documents

### ISSUE THREAD

**Issue by gbuehler** [metanorma/metanorma-ogc#195]
When we are producing an "Approved" document, meaning that it is no longer a draft, the "Warning for Drafts" is still being included.

![Screen Shot 2020-10-02 at 9 51 14 AM](https://user-images.githubusercontent.com/221625/94931012-3b82d400-0495-11eb-9514-cc552c25be87.png)

Only if I change the status to "Published" do I get the correct "Warning"

![Screen Shot 2020-10-02 at 9 53 24 AM](https://user-images.githubusercontent.com/221625/94931025-40e01e80-0495-11eb-847c-7f444442fba2.png)

**Comment by opoudjis** [comment]
At this time, "Approved" is not recognised as a valid document stage in Metanorma for OGC. The stages it recognises are:

https://www.metanorma.com/author/ogc/authoring/

````
swg-draft: SWG Draft. This is the draft created after the TC approval and PC approval votes.
oab-review: OAB Review. This is the intended draft for the “OAB + OGC-NA Review”.
public-rfc: Public RFC. This is the draft for the (30 days) public comment period.
tc-vote: TC Vote. This is the intended draft for the TC adoption and PC adoption votes.
published: Published. This is the document intended to be published, after comments are handled with the TC chair (after tc-vote).
deprecated: Deprecated. This document has been deprecated.
retired: Retired. This document has been retired and no longer considered normative.
````

And only the last three are deemed not to be drafts, and do not occur with the drafts warning.

Is "Approved" a distinct status from "Published"? Are there other document statuses that are not on the list?

**Comment by ronaldtse** [comment]
This has been implemented in #187 and documentation will be updated in #196. Closing for now. Thanks @gbuehler !
